tools/runners/test262-harness.py

```diff
@@ -595,11 +600,15 @@ def execute(command):
             logging.info("exec: %s", str(args))
             process = subprocess.Popen(
                 args,
-                shell=is_windows(),
+                shell=False,
                 stdout=stdout.file_desc,
                 stderr=stderr.file_desc
             )
+            timeout = TEST262_CASE_TIMEOUT
+            timer = threading.Timer(timeout, process.kill)
```

Comment:
these two lines can go in a single one (no need for the `timeout` variable when `TEST262_CASE_TIMEOUT` is already there)

Reply:
for future usage and clear diff.
If you insist I revert it

Reply:
The diff would be just as clear with `timer = threading.Timer(TEST262_CASE_TIMEOUT, process.kill)`. I'm not sure about future use cases. If and when they kick in, it will be trivial to split this into two lines. To be reviewed at that point of time.

Reply:
> The diff would be just as clear with `timer = threading.Timer(TEST262_CASE_TIMEOUT, process.kill)`. I'm not sure about future use cases. If and when they kick in, it will be trivial to split this into two lines. To be reviewed at that point of time.

done